### PR TITLE
[SP-7704] Add actions for IDFA

### PR DESCRIPTION
**Pull request by vilasma**
Added `IDFAAccepted` & `IDFADenied` to `SPActionType` to load the Messages once received IDFA status from user

### CODE REVIEW

File: Example/Podfile [hunk 1]
```diff
@@ -40,12 +40,6 @@ target 'NativeMessageExampleUITests' do
   pod 'Nimble'
 end
 
-target 'SourcePointMetaApp' do
-  pod 'ConsentViewController', :path => '../'
-  pod 'IQKeyboardManagerSwift'
-  pod 'Wormholy'
-end
-
```
[the remark this answers is not in the document]
Comment (vilasma) [reply Example/Podfile:48]: As we have only UI tests for Meta app not unit tests so I removed the Unit test target from Podfile

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# 6.1.6 (August, 16, 2021)
+* Added `IDFAAccepted` & `IDFADenied` to `SPAction` to load the messages based on IDFA status [#330](https://github.com/SourcePointUSA/ios-cmp-app/pull/330)
```
[the remark this answers is not in the document]
Comment (vilasma) [reply CHANGELOG.md:2]: Will update the Changelog file

File: Example/Podfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (andresilveirah) [reply Example/Podfile:48]: @vilasma the removed target was `SourcePointMetaApp` not the tests one (`SourcePointMetaAppTests`)

[the remark this answers is not in the document]
Comment (vilasma) [reply Example/Podfile:48]: Yes @andresilveirah, removed target was `SourcePointMetaApp`. 
Now I have added `SourcePointMetaApp` target and removed `SourcePointMetaAppTests` target

**Pull request merged**